paper-fab.html

```diff
@@ -103,12 +108,14 @@
       iron-icon {
         @apply(--paper-fab-iron-icon);
       }
-      
+
       span {
         width: 100%;
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        text-align: center;
+        @apply(--paper-fab-label);
```

Comment:
I don't think the label would be a good insertion point for layout properties - if you add margin/padding to the label, the label could get cut off because `:host` sets explicit width/height.

Looking back at the mixin for the icon (#28), I would want to see a good use case for the mixin before adding it. I think one of the advantages of style scoping is that the element author can protect implementation details and only allow certain properties to be set (until such a point where there are so many of these properties that it become impractical to declare all of them and a mixin becomes an acceptable compromise). Also, adding a new mixin also means it becomes more difficult to change the API in the future, if that ever becomes the case.

I'll leave the choice up to you though, just wanted to drop my two cents :)